derive_builder_core/src/setter.rs

```diff
@@ -160,10 +160,15 @@ impl<'a> ToTokens for Setter<'a> {
                 let try_ident = syn::Ident::new(&format!("try_{}", ident), Span::call_site());
 
                 let mut converted = quote! {converted};
+
```

Comment:
Nit: I'd remove this whitespace, and the whitespace on line 167 (or maybe even put all of this inside one block):

```rust
let converted = {
    let mut converted = ...;
    // ... if blocks...
    converted
};
```

I'm not sure why I didn't do that before...